src/js/modules/infragistics.ui.shared.js

```diff
@@ -460,6 +460,9 @@
 
 		clearBookmarks: function () {
 			if (this.bookmarks) {
+
+				//V.S. February 16, 2018 #1566 VideoPlayer destroy removes the tooltip correctly.
+				$("#" +  this._id("_tooltip")).igTooltip("destroy").remove();
```

Comment:
[remove() ](https://stackoverflow.com/questions/15939503/does-jquery-remove-also-destroy-resizables-draggables) calls destroy() method automatically, so you don't need to call destroy manually. Thanks!